packages/web-components/src/components/va-icon/va-icon.tsx

```diff
@@ -41,14 +49,21 @@ export class VaIcon {
     return this.size;
   }
 
+  componentWillLoad() {
+    if (!globalThis.getVaIconSpriteLocation) {
```

Comment:
Call init, imported from the new global function to get the default `/img/sprite.svg` path if init has not been called previously